### ISSUE TITLE
[UNet_Medical/TensorFlow2] Display validation score after each epoch

### ISSUE THREAD

**Issue by gherbox** [NVIDIA/DeepLearningExamples#710]
I am running ./examples/unet_1GPU.sh and ./examples/unet_8GPU.sh
All the accuracy results that I obtain are the results on the trainset. The accuracy results for the validation set are only displayed at the end of the training.

It would be more insightful to get the accuracy results for the validation set after each epoch, so we can compare the performance of distributed learning on the accuracy results on the validation set and not the train set.

**Comment by mmarcinkiewicz** [comment]
Thanks @gherbox for this comment. We are looking into that.

**Comment by mmarcinkiewicz** [comment]
Hi @gherbox . I've added the requested functionality. It is not added to the prepared scripts because it will extend the time to train and it might not be desirable for everyone.

In order to enable it for your usecase please use the full command to train, as in the point 5 of the [quick-start-guide](https://github.com/NVIDIA/DeepLearningExamples/tree/master/TensorFlow2/Segmentation/UNet_Medical#quick-start-guide) with additional parameter `--evaluate_every n` where `n` is the evaluation interval. 

You can use `--log_every` with the same `n` to get the training loss and evaluation loss every `n` iterations.

Closing the issue. Please reopen of you have any more problems.
